third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -1144,12 +1169,13 @@ void OpenGLRenderer3D::RenderVertexBuffer(
   DO_VALIDATION;
   VertexBuffer *vertexBuffer;
 
-  if (renderMode != e_RenderMode_GeometryOnly) {
-    DO_VALIDATION;
-    mapping.glEnable(GL_TEXTURE_2D);
-  } else {
-    mapping.glDisable(GL_TEXTURE_2D);
-  }
+// Deprecated
+//  if (renderMode != e_RenderMode_GeometryOnly) {
+//    DO_VALIDATION;
+//    mapping.glEnable(GL_TEXTURE_2D);
+//  } else {
+//    mapping.glDisable(GL_TEXTURE_2D);
+//  }
```

Comment:
Same as above
https://stackoverflow.com/questions/14179497/is-it-necessary-to-call-glenablegl-texture-before-using-textures-in-opengl-2-1